### ISSUE TITLE
Cited video found irrelevant

### ISSUE THREAD

**Issue by Kumaraditya03** [dwyl/learn-hapi#122]
Hey Team, I went on to follow the repo for getting some knowledge about hapi, but looks like the video cited (code winds)  is outdated in today's context. Could you please update the content?

**Comment by nelsonic** [comment]
@Kumaraditya03 agreed. It’s been a few years and the video is no longer relevant. Thanks for opening this issue to inform.
if you have time, please create a Pull Request and remove it. 👍🏻
